.github/ISSUE_TEMPLATE/bug.yml

```
@@ -50,19 +50,19 @@ body:
     attributes:
       description: >-
         Copy-paste the `flake.lock` file. If flakes are not used, provide the
-        locked input versions.
+        locked input versions. Do not remove any pre-filled text in the field.
 
       label: flake.lock
 
-      placeholder: |-
-        {
-          "nodes": {
-          },
-          "root": "root",
-          "version": 7
-        }
-
-      render: json
+      value: |-
+        <details>
+        <summary>flake.lock</summary>
```

What about formatting as monospace:

```suggestion
        <summary><code>flake.lock</code></summary>
```